src/Features/LanguageServer/Protocol/Features/Options/DiagnosticOptionsStorage.cs

```diff
@@ -4,17 +4,16 @@
 
 using Microsoft.CodeAnalysis.Options;
 
-namespace Microsoft.CodeAnalysis.Diagnostics
+namespace Microsoft.CodeAnalysis.Diagnostics;
+
+internal sealed class DiagnosticOptionsStorage
 {
-    internal sealed class DiagnosticOptionsStorage
-    {
-        public static readonly Option2<bool> LspPullDiagnosticsFeatureFlag = new(
-            "dotnet_enable_lsp_pull_diagnostics", defaultValue: false);
+    public static readonly Option2<bool> PullDiagnosticsFeatureFlag = new(
+        "dotnet_enable_pull_diagnostics", defaultValue: true);
```

Comment:
renaming this I think will reset it for people who previously changed the value.  Fine with that since we're switching the default, but FYI.

Reply:
yup.  reverting this.